client/my-sites/purchases/billing-history/index.tsx

```diff
@@ -76,6 +76,7 @@ export function BillingHistory( { siteSlug }: { siteSlug: string } ) {
 				brandFont
 				className="billing-history__page-heading"
 				headerText={ titles.sectionTitle }
+				subHeaderText={ translate( 'View, print, and email your receipts.' ) }
```

Comment:
Since this is the site-specific view, can we update it to `View, print, and email your receipts for this site.`

Reply:
Instead of `View, print, and email your receipts for this site.`, should we go as far as `View, print, and email your receipts for example.com` with the specific site address (and replicate to the other site-specific views)?

Reply:
I think that's a good idea, but I'm worried about any confusion that might be attached to multiple domains on a single site, or for instance if an email was for a secondary domain. Customer Home and Domains both use "your site", which I thought about doing, but "email _your_ receipts for _your_ site" felt redundant.

Reply:
This is now added.

<img width="490" alt="Screen Shot 2021-07-23 at 12 33 12 PM" src="https://user-images.githubusercontent.com/8002138/126832502-648f753e-c7a8-4085-a4a7-6cc633cc5266.png">

Marking as resolved.